### DISCUSSION TITLE
Gap between navbar and content in single layout pages

### DISCUSSION THREAD

**Discussion by ativarovsky** [mmistakes/minimal-mistakes#2661] [Q&A] [answered]
Hi there, 

I've been up and running with the theme for over a month and it's fantastic.  Last week I opened my site and all the pages using the single layout have a huge gap between the navbar and the content. It's only there on full-screen, disappears when I reduce the size. I've been playing with the _page.scss for about 4 days and haven't been able to fix it. Not sure what caused the change but running bundle update didn't work. 

Any thoughts on why this is happening would be hugely appreciated. I'm very much a newb to site coding. 

Thanks in advance.

**Comment by iBug** [comment]
Can you provide an example page and the source code repository so we can investigate? Right now we have no idea what to look at.

**Comment by ativarovsky** [comment]
Yep, this is the source code repository: https://github.com/ativarovsky/ativarovsky.github.io

The example page is: https://ativarovsky.github.io/_pages/about/

**Comment by iBug** [comment, marked as answer]
You tweaked the CSS yourself which led to the result you're seeing.

<https://github.com/ativarovsky/ativarovsky.github.io/commit/9b83fe9d535a2dcc9e556f6b164b4b419949c054#diff-b4b598c232ce5e79fb5dcf6dfdd289b0R46>

Either revert this change, or tweak the line right under as well:

```diff
-padding-right: $right-sidebar-width;
+padding-right: $right-sidebar-width-narrow;

**Comment by ativarovsky** [comment]
oh man...  ok, thanks so much for your help.
